src/activity.ts

```diff
@@ -50,7 +44,18 @@ export function onDiagnosticsChange() {
 	totalProblems = counted;
 }
 
-export function activity(previous: Presence = {}, isViewing = false): Presence {
+async function getFileURI() {
+	// temporary workaround to get the file URI of the current active CustomTextEditor
+	// https://github.com/microsoft/vscode/issues/3553#issuecomment-1098562676
+
+	const origClipboard = await env.clipboard.readText();
+	await commands.executeCommand('copyFilePath');
+	const filePath = await env.clipboard.readText();
+	await env.clipboard.writeText(origClipboard);
+	return Uri.file(filePath);
+}
```

Comment:
Indirectly obtains the file's URI, which would then be used for resolving the file's icon if the `window.activeTextEditor` object is `undefined`.